src/components/Login.js

```diff
@@ -40,42 +40,39 @@ class Login extends Component {
 
   render () {
     return (
-      <TransitionGroup component="div"
-        transitionName="login-transition"
-        transitionAppear
-        transitionAppearTimeout={0}
-        transitionEnterTimeout={0}
-        transitionLeaveTimeout={0}>
-        <FramelessLayout className={cx('rka-login-page', {'login-transition-leave': this.state.showTransition})} {...this.props}>
-          <div className="rka-login-container">
-            <div className="row">
-              <div className="col-md-5">
-                <div className="rka-login-brand">
-                  <i className="rka-header-logo mb-lg" />
-                  <h2 className="txt-white rka-h2">Web images done right.</h2>
-                  <p className="rka-login-brand-powered">Powered by Liip.</p>
+      <TransitionGroup>
+        <CSSTransition appear classNames="login-transition" timeout={0}>
```

Comment:
@pedro-couto we'll need to double check this and the other transition group changes and see if it still works as expected.
Docs:
https://github.com/reactjs/react-transition-group
https://github.com/reactjs/react-transition-group/blob/master/Migration.md
https://reactcommunity.org/react-transition-group/#Transition